css-alpha-channel/index.html

```diff
@@ -0,0 +1,79 @@
+---
+feature_name: Alpha Channel Support in CSS Color Parsing
+chrome_version: 52
+feature_id: 5685348285808640
+---
+
+<h3>Background</h3>
+<p>Starting with Chrome 52, CSS rules now accept an alpha (transparency) channel for color values. Do this by specifying a fourth number in contexts that take an RGB value. Alpha channel support takes several forms.</p>
```

Comment:
The background is misleading—Chrome has supported alpha channels for a _long_ time (at least [going back to 2010](https://css-tricks.com/rgba-browser-support/#browser-support)), but previously only using the `rgba()` syntax. The change in M52 is to add support for the hex syntax.